README.md

```diff
@@ -1,2 +1,113 @@
-# OOP-school-library
-A library tool to record what books are in the library and who borrows them.
+<a name="readme-top"></a>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 OOP-school-library <a name="morse-decoder"></a>
+
+> A library tool to record what books are in the library and who borrows them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Rubocop](https://docs.rubocop.org/rubocop/1.39/index.html)
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Records books
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ Visual Studio Code
+ git bash
+ Ruby
+```
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/MarcoOquendoC/OOP-school-library.git
+```
+### Install
+
+Install this project with:
+
+```sh
+  cd OOP-school-library
+  bundle install
+```
+### Usage
+
+To run the project, execute the following commands:
+
+```sh
+  irb
+  irb(main):001:0> require "./person.rb"
+  irb(main):001:0> require "./student.rb"
+  irb(main):001:0> require "./teacher.rb"
+```
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+### 👤 Marco Oquendo 
+
+- GitHub: [@MarcoOquendoC](https://github.com/MarcoOquendoC). 
+- Twitter: [@marcooquendoc](https://twitter.com/marcooquendoc). 
+- LinkedIn: [Marco Vinicio Oquendo](https://www.linkedin.com/in/MarcoOquendoC/). 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Your link to the issues page is broken 😨 Kindly fix it to make your readme file descriptive 👍